ssl.go

```diff
@@ -16,9 +15,9 @@ type ZoneCustomSSL struct {
 	Status        string     `json:"status"`
 	BundleMethod  string     `json:"bundle_method"`
 	ZoneID        string     `json:"zone_id"`
-	UploadedOn    time.Time  `json:"uploaded_on"`
```

Comment:
When I tested this, the format of the time string returned from Cloudflare ("2016-07-06 21:27:40.124575+00") did not match what was mentioned in the API docs ("2014-01-01T05:20:00Z"), causing this parsing to fail.